openreview/venue/venue.py

```diff
@@ -1075,8 +1075,11 @@ def ithenticate_create_and_upload_submission(self):
 
         submissions = self.get_submissions()
         for submission in tqdm(submissions):
-            # Create a new submission ID in iThenticate
             # TODO - Decide what should go in metadata.group_context.owners
+            if (self.client.get_edges_count(head=submission.id, invitation=self.get_iThenticate_plagiarism_check_invitation_id()) > 0 and 
```

Comment:
I think making two API calls for every submissions can be too expensive and the process will be very slow for 10k.

You can load all the edges in a dictionary before starting the loop and check the label status using this dictionary instead of doing an API call.

Also my preference is to avoid the sentence `continue` in a loop.

Reply:
Sure, I'll make that change

Reply:
I've removed the multiple API calls, loaded the edges in a dictionary, and removed the `continue` as well